test_players/load_engine.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2022 Liberty Global B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+"use strict";
+
+var player_type = getQueryVariable("player");
+var player_ver = getQueryVariable("player_ver");
+var media_url = getQueryVariable("url");
+if (!media_url) {
+  media_url = "https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd";
```

Comment:
Set one of our streams as default e.g. test-materials/dash/fmp4_h264_aac/manifest.mpd

Reply:
done